src/tests/ftest/container/snapshot.py

```diff
@@ -205,7 +205,8 @@ def test_snapshot_negativecases(self):
                 (7)Verify snap_list bad parameter behavior.
 
         Use Cases: Combinations with minimun 1 client and 1 server.
-        :avocado: tags=snap,snapshot_negative,snapshotcreate_negative
+        :avocado: tags=all,small,smoke,pr,snap,snapshot_negative,
+                       snapshotcreate_negative
```

Comment:
Does breaking this up into multiple lines actually work?  Have you tried using the avocado filter feature and this `snapshotcreate_negative` tag to ensure that this test is discovered?

Reply:
Done